### ISSUE TITLE
Server 500 error

### ISSUE THREAD

**Issue by mvnixon** [alexjustesen/speedtest-tracker#813]
**Describe the bug**
Whenever I access the web gui, I now get a server 500 error. Logs show 'Error: No Application encryption key has been specified' with a backtrace (Logs below). Seems to have started on v0.12.2

**To Reproduce**
Access Web GUI,
Met with Server 500 error
Logs

**Environment (please complete the following information):**
 - OS: Synology
-  Architecture: amd64
 - Browser safari
 - Version 0.12.2

**Logs**
`
2023/09/23 08:48:59 | stdout | 172.19.0.1 - - [23/Sep/2023:08:48:59 +0200] "GET /api/speedtest/latest HTTP/1.1" 200 397 "-" "-"
2023/09/23 08:48:24 | stdout | 172.19.0.1 - - [23/Sep/2023:08:48:24 +0200] "GET /api/speedtest/latest HTTP/1.1" 200 397 "-" "-"
2023/09/23 08:48:02 | stderr | "}
2023/09/23 08:48:02 | stderr | #19 {main}
2023/09/23 08:48:02 | stderr | #18 /var/www/html/public/index.php(55): Illuminate\\Foundation\\Http\\Kernel->terminate()
2023/09/23 08:48:02 | stderr | #17 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(213): Illuminate\\Foundation\\Http\\Kernel->terminateMiddleware()
2023/09/23 08:48:02 | stderr | #16 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(255): Illuminate\\Foundation\\Application->make()
2023/09/23 08:48:02 | stderr | #15 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(922): Illuminate\\Container\\Container->make()
2023/09/23 08:48:02 | stderr | #14 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\\Foundation\\Application->resolve()
2023/09/23 08:48:02 | stderr | #13 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(937): Illuminate\\Container\\Container->resolve()
2023/09/23 08:48:02 | stderr | #12 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\\Container\\Container->build()
2023/09/23 08:48:02 | stderr | #11 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(943): Illuminate\\Container\\Container->resolveDependencies()
2023/09/23 08:48:02 | stderr | #10 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(982): Illuminate\\Container\\Container->resolveClass()
2023/09/23 08:48:02 | stderr | #9 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(1066): Illuminate\\Foundation\\Application->make()
2023/09/23 08:48:02 | stderr | #8 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(922): Illuminate\\Container\\Container->make()
2023/09/23 08:48:02 | stderr | #7 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(731): Illuminate\\Foundation\\Application->resolve()
2023/09/23 08:48:02 | stderr | #6 /var/www/html/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(937): Illuminate\\Container\\Container->resolve()
2023/09/23 08:48:02 | stderr | #5 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(795): Illuminate\\Container\\Container->build()
2023/09/23 08:48:02 | stderr | #4 /var/www/html/vendor/laravel/framework/src/Illuminate/Container/Container.php(908): Illuminate\\Encryption\\EncryptionServiceProvider->Illuminate\\Encryption\\{closure}()
2023/09/23 08:48:02 | stderr | #3 /var/www/html/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(32): Illuminate\\Encryption\\EncryptionServiceProvider->parseKey()
2023/09/23 08:48:02 | stderr | #2 /var/www/html/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(60): Illuminate\\Encryption\\EncryptionServiceProvider->key()
2023/09/23 08:48:02 | stderr | #1 /var/www/html/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php(77): tap()
2023/09/23 08:48:02 | stderr | #0 /var/www/html/vendor/laravel/framework/src/Illuminate/Support/helpers.php(307): Illuminate\\Encryption\\EncryptionServiceProvider->Illuminate\\Encryption\\{closure}()
2023/09/23 08:48:02 | stderr | [stacktrace]
2023/09/23 08:48:02 | stderr | [2023-09-23 08:48:02] production.ERROR: No application encryption key has been specified. {"exception":"[object] (Illuminate\\Encryption\\MissingAppKeyException(code: 0): No application encryption key has been specified. at /var/www/html/vendor/laravel/framework/src/Illuminate/Encryption/EncryptionServiceProvider.php:79)
2023/09/23 08:48:02 | stdout | 172.19.0.1 - - [23/Sep/2023:08:48:02 +0200] "GET / HTTP/1.1" 500 6628 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.0 Safari/605.1.15"
2023/09/23 08:48:02 | stderr
`

**Comment by alexjustesen** [comment]
App key is generated on startup now so this is likely a caching issue. You can follow these steps from the docs to resolve it. 

https://docs.speedtest-tracker.dev/faqs#i-get-a-warning-on-container-start-up-that-the-app_key-is-missing

**Comment by Rahzadan** [comment]
I had this exact issue when I tried to access my speedtest-tracker web gui. @alexjustesen 's suggestion above did not work for me.

What did fix it for me however, is follow the directions here: https://docs.speedtest-tracker.dev/faqs#database - I was using SQLite since the beginning, so I believe it could suddenly not see the DB. Adding the suggested env variables to my docker-compose file still didn't fix it, so I also deleted the generated **/config/.env** file, which forced speedtest-tracker to regenerate it at startup. Stopped and restarted speedtest-tracker. Problem fixed!
